packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -12,33 +38,52 @@ import { useFocusableGroup } from '@fluentui/react-tabster';
  * @param props - props from this instance of Card
  * @param ref - reference to root HTMLElement of Card
  */
-export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>): CardState => {
-  const { appearance = 'filled', focusMode = 'off', orientation = 'vertical', size = 'medium' } = props;
+export const useCard_unstable = (props: CardProps, ref: React.Ref<CardRefElement>): CardState => {
+  const { appearance = 'filled', orientation = 'vertical', size = 'medium', as = 'div' } = props;
+  const cardRef = React.useRef<CardRefElement>(null);
 
-  const focusMap = {
-    off: undefined,
-    'no-tab': 'limited-trap-focus',
-    'tab-exit': 'limited',
-    'tab-only': 'unlimited',
-  } as const;
+  const { isSelectable, hasSelectSlot, selectableProps, selectableSlot, isCardSelected } = useCardSelectable(
+    props,
+    cardRef,
+  );
 
-  const groupperAttrs = useFocusableGroup({
-    tabBehavior: focusMap[focusMode],
-  });
+  const isInteractive = Boolean(
+    isSelectable ||
+      ['a', 'button'].includes(as) ||
+      props.onClick ||
+      props.onDoubleClick ||
+      props.onMouseUp ||
+      props.onMouseDown ||
+      props.onPointerUp ||
+      props.onPointerDown ||
+      props.onTouchStart ||
+      props.onTouchEnd,
+  );
 
-  const focusAttrs = focusMode !== 'off' ? { tabIndex: 0, ...groupperAttrs } : null;
+  const focusAttributes = useCardFocusAttributes(props, isInteractive);
 
   return {
     appearance,
     orientation,
     size,
+    isInteractive,
+    isSelectable,
+    hasSelectSlot,
+    isCardSelected,
+
+    components: {
+      root: as,
+      select: 'div',
+    },
 
-    components: { root: 'div' },
-    root: getNativeElementProps(props.as || 'div', {
-      ref,
+    root: getNativeElementProps(as, {
+      ref: ref || cardRef,
       role: 'group',
-      ...focusAttrs,
+      ...focusAttributes,
       ...props,
+      ...selectableProps,
```

Comment:
The actual card is focusable and can be interacted with to select the card, but there is no other accessible markup other than `aria-checked` on the selection slot. Is it clear for screen reader users that cards are selectable ?

Reply:
I changed this behavior in a recent commit. All other accessibility changes will be covered by a follow-up PR, which will also better document cards with a11y props.